astacus/common/cassandra/schema.py

```diff
@@ -97,6 +97,12 @@ def __lt__(self, o: CassandraNamed) -> bool:
             return True
         return False
 
+    @staticmethod
```

Comment:
nit: staticmethods should be used sparingly (as this doesn't really need the class or instance, it could be just separate function)

Reply:
Done